frontend/src/components/deleteUserModal/index.tsx

```diff
@@ -10,15 +10,14 @@ import { StaffData } from '../../constants/employees';
 
 import { deleteStaff } from '../../features/staffSlice';
 import 'toastr/build/toastr.min.css';
-import employeesStyle from './employees.module.scss';
+import deleteUserModalStyle from './deleteUserModal.module.scss';
 
 interface Props {
-  isOpen: boolean;
   staff: StaffData;
   closeModalFunction: () => void;
```

Comment:
I believe closeModalFunction is not used anymore, because modal's state is handled by redux?